tfhe/src/shortint/parameters/compact_public_key_only/p_fail_2_minus_64/ks_pbs.rs

```diff
@@ -6,13 +6,26 @@ use crate::shortint::parameters::{
     MessageModulus,
 };
 
+/// This parameter set should be used when doing zk proof of public key encryption
 pub const PARAM_PKE_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64: CompactPublicKeyEncryptionParameters =
     CompactPublicKeyEncryptionParameters {
-        encryption_lwe_dimension: LweDimension(1024),
-        encryption_noise_distribution: DynamicDistribution::new_t_uniform(42),
+        encryption_lwe_dimension: LweDimension(2048),
+        encryption_noise_distribution: DynamicDistribution::new_t_uniform(17),
         message_modulus: MessageModulus(4),
         carry_modulus: CarryModulus(4),
         ciphertext_modulus: CiphertextModulus::new_native(),
         expansion_kind: CompactCiphertextListExpansionKind::RequiresCasting,
     }
     .validate();
+
+/// This legacy parameter set should be used with the v1 pke zk scheme
+pub const PARAM_PKE_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64_ZK_V1:
```

Comment:
also this does not respect the potential TFHE-rs version being in the name of consts

Reply:
I know, I did not reworked the params yet